src/devices/generic_temperature_sensor.c

```diff
@@ -5,16 +5,25 @@
  * it under the terms of the GNU General Public License as published by
  * the Free Software Foundation; either version 2 of the License, or
  * (at your option) any later version.
+ *
+ * Switched to new data API by 'ovrheat' Nik
  */
```

Comment:
I am not so found of these type of changes. This info can be found in the git repository log. If you feel that you contributed significantly then add your name as a copyright line.